back/database/factories/MessageFactory.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Models\Message;
+use Faker\Generator as Faker;
+
+$factory->define(Model::class, function (Faker $faker) {
+    return [
+      'title' => $faker->text(10),
+      'content' => $faker->sentence,
+    ];
+});
```

Comment:
Là ça devrait marcher 🙃

```suggestion
use App\Models\Message;
use Faker\Generator as Faker;

$factory->define(Message::class, function (Faker $faker) {
    return [
      'title' => $faker->text(10),
      'content' => $faker->sentence,
    ];
});
```